crates/stages/src/stages/tx_lookup.rs

```diff
@@ -92,43 +96,71 @@ impl<DB: Database> Stage<DB> for TransactionLookupStage {
             return Ok(ExecOutput::done(input.checkpoint()))
         }
 
-        let (tx_range, block_range, is_final_range) =
-            input.next_block_range_with_transaction_threshold(provider, self.commit_threshold)?;
-        let end_block = *block_range.end();
+        // 500MB temporary files
+        let mut hash_collector: Collector<TxHash, TxNumber> =
+            Collector::new(Arc::new(TempDir::new()?), 500 * (1024 * 1024));
```

Comment:
any reason to make buffer capacity configurable here?

Reply:
related to https://github.com/paradigmxyz/reth/issues/6696, but should make things slightly faster on bigger sized chunks